docs/extensions/installing-extensions.md

```diff
@@ -0,0 +1,139 @@
+---
+contributors: Esther Agbaje
+description: Discover how to install extensions into your Directus project.
+---
+
+# Installing Extensions
+
+There are two possible ways to install extensions to your Directus instance:
+
+1. Installing through NPM
+2. Installing through the Extensions Folder
+
+## Installing through NPM
```

Comment:
Should we add a note here that this installation method makes sense if you want to install an existing public extension or if have published (or plan to do so) your own extension on npm?